cmd/ui/src/ducks/explore/reducer.ts

```diff
@@ -45,6 +46,8 @@ const graphDataReducer = (state = initialGraphDataState, action: types.GraphActi
             // handle delete nodes (action.ids)
         } else if (action.type === types.GRAPH_INIT) {
             draft.init = action.payload;
+        } else if (action.type === types.SAVE_RESPONSE_FOR_EXPORT) {
+            draft.export = action.payload;
```

Comment:
Out of curiosity - why do we want to store data for export separately? Can we not use whatever the current chartProps/items are to populate the JSON file to be exported?

Reply:
the data that we export is supposed to be a 1:1 copy of the response we get from the API.  we cant use chartProps for this, because the data that gets saved to chartProps has been massaged into a new shape for rendering purposes, so I added the "export" field as the place where we can save that raw API response.

Reply:
That makes sense - thank you for the clarification.

Reply:
Depending on the specific API call being made the resulting JSON structure could be quite different - is that accurate?

Reply:
So based on our internal conversation we decided that the desired end state is to transform our JSON before producing a file so that the exported files have a consistent structure.